2017/CG-05.md

```diff
@@ -51,9 +51,13 @@ Please register before the event [here](https://goo.gl/forms/LQVhuRuI22lK8Umw1).
        * [Proposal](https://github.com/WebAssembly/threads/blob/master/proposals/threads/Overview.md)
        * Brief overview of the proposal
        * Topics for committee feedback
+         - [Matching SharedArrayBuffer/Atomics for v1](https://github.com/WebAssembly/threads/issues/5)
+         - [Blocking main thread in web embedding](https://github.com/WebAssembly/threads/issues/6)
+         - [float atomics, i64 atomics & i64.wait](https://github.com/WebAssembly/threads/issues/7)
```

Comment:
Poll these separately. It's OK for the issue, but for polling you want people to be able to express themselves differently here.